### ISSUE TITLE
Broken link on README

### ISSUE THREAD

**Issue by pilotcam** [FactoryTalk-Optix/FeaturesDemo#1]
The link [in this section](https://github.com/FactoryTalk-Optix/FeaturesDemo#multiple-plcs)  that should point to PLC files is broken.

**Comment by InfiniteReaction** [comment]
It is still broken?

**Comment by camschollrok** [comment]
Yes - still broken for me.  It appears to point to a project in a private organization.
